schema/encoding/jsonschema/all_go16_test.go

```diff
@@ -0,0 +1,11 @@
+// +build !go1.7
+
+package jsonschema_test
```

Comment:
Excellent, thanks for explaining. The encoding test was testing imports rather than public interface so the intention was different. I was ignorant of this pattern.